### ISSUE TITLE
Cannot seem to send text

### ISSUE THREAD

**Issue by meldra** [LithApp/Lith#115]
Android 12

Return key just seems to add a newline

**Comment by meldra** [comment]
nevermind the send button is just disabled by default for some reason?

**Comment by MartinBriza** [comment]
Well, that's still a bug, this works on iOS. I'll reopen this. It's probably caused by some other touch area obscuring the button.

**Comment by meldra** [comment]
The configuration for for "show send button" is off by default on the android build. Is there is supposed to be some way of sending that isn't the return/enter key?

The return key adds non-visible non-space content that I expect is newlines. The text preview doesn't render the newlines, but i have to backspace the same number of times i hit return to be able to backspace the text i tried to send.

**Comment by chinese-soup** [comment]
> Is there is supposed to be some way of sending that isn't that or the return/enter key?
Nope, just that button and keyboard's Enter key (which currently only working on desktop and iOS, as mentioned).

> The return key adds non-visible non-space content that I expect is newlines.
Yep. The newlines not being rendered and being just non-visible non-space are a known issue, since posting multiline is only implemented in the backend atm: https://github.com/LithApp/Lith/issues/29#issuecomment-939494240
